src/main/views/viewManager.ts

```diff
@@ -70,15 +72,17 @@ export class ViewManager {
         MainWindow.on(MAIN_WINDOW_FOCUSED, this.focusCurrentView);
         ipcMain.handle(GET_VIEW_INFO_FOR_TEST, this.handleGetViewInfoForTest);
         ipcMain.handle(GET_IS_DEV_MODE, () => isDev);
+        ipcMain.handle(REQUEST_BROWSER_HISTORY_STATUS, this.handleRequestBrowserHistoryStatus);
         ipcMain.on(HISTORY, this.handleHistory);
         ipcMain.on(REACT_APP_INITIALIZED, this.handleReactAppInitialized);
         ipcMain.on(BROWSER_HISTORY_PUSH, this.handleBrowserHistoryPush);
-        ipcMain.on(BROWSER_HISTORY_BUTTON, this.handleBrowserHistoryButton);
         ipcMain.on(APP_LOGGED_IN, this.handleAppLoggedIn);
         ipcMain.on(APP_LOGGED_OUT, this.handleAppLoggedOut);
         ipcMain.on(RELOAD_CURRENT_VIEW, this.handleReloadCurrentView);
-        ipcMain.on(UNREAD_RESULT, this.handleFaviconIsUnread);
+        ipcMain.on(UNREAD_RESULT, this.handleUnreadChanged);
+        ipcMain.on(MENTIONS_RESULT, this.handleMentionsChanged);
         ipcMain.on(SESSION_EXPIRED, this.handleSessionExpired);
+        ipcMain.on(LEGACY_OFF, this.handleLegacyOff);
```

Comment:
We also have the preload tell us when we're using the new API to turn off some listeners in the Main Process as well.